Project.toml

```diff
@@ -13,6 +13,9 @@ Millboard = "39ec1447-df44-5f4c-beaa-866f30b4d3b2"
 OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
+SafeTestsets = "1bc83da4-3b8d-516f-aca4-4fe02f6d838f"
```

Comment:
The testing dependencies need to be added to a separate Project.toml file inside `test/`

Reply:
Ok. Then I prepare the tests from within the "tests"-folder and will document this in the right place.